### PR TITLE
Update tables for the remaining components after confirming format

### PR DESCRIPTION
**Pull request by isabelle-dr**
This PR updates the tables that have the list of files and fields included on the individual component pages

### CODE REVIEW

File: docs/documentation/schedule/components/base.md [hunk 1]
```diff
@@ -4,73 +4,53 @@ All these features must be implemented together to enable a working GTFS feed.
...
 Agencies contain basic information about the agencies responsible for the transit service, such as their name, website URL, and the language and timezone in which the service operates. This allows to match specific services with their corresponding agency.
 
-| Files associated      | [Agency.txt](/schedule/reference/#agencytxt)                                                                                                     |
-|-----------------------|----------------------------------------------------------------------------------------------------------------|
-| **Fields associated** | agency_name<br>agency_url<br>agency_timezone<br>agency_lang<br>agency_phone<br>agency_fare_url<br>agency_email |
-
-</div>
+| Files included                   | Fields included   |
+|----------------------------------|-------------------|
+|[agency.txt](/schedule/reference/#agencytxt)|`agency_name`, `agency_url`, `agency_timezone`, `agency_lang`, `agency_phone`, `agency_fare_url`, `agency_email |
```
Comment (tzujenchanmbd) [inline docs/documentation/schedule/components/base.md]:
```suggestion
|[agency.txt](/schedule/reference/#agencytxt)|`agency_name`, `agency_url`, `agency_timezone`, `agency_lang`, `agency_phone`, `agency_fare_url`, `agency_email` |
```

File: docs/documentation/schedule/components/base.md [hunk 2]
```diff
@@ -4,73 +4,53 @@ All these features must be implemented together to enable a working GTFS feed.
...
 Certain services operate on regular a frequency (e. g. a subway line that runs every 5min) rather than having specific arrival and departure times. This can be modeled using Frequency-based sercices (@TODO link), and this can be modeled in conjunction with `stop_times.txt`.
 
-| Files associated      | [stop_times.txt](/schedule/reference/#stop_timestxt)                                                                                                                            |
-|-----------------------|-------------------------------------------------------------------------------------------------------------------------------------------|
-| **Fields associated** | trip_id<br>arrival_time<br>departure_time<br>stop_id<br>stop_sequence<br>pickup_type<br>drop_off_type<br>timepoint |
-
-</div>
+| Files included                   | Fields included   |
+|----------------------------------|-------------------|
+|[stop_times.txt](/schedule/reference/#stop_timestxt)|`trip_id`, `arrival_time`, `departure_time`, `stop_id`, `stop_sequence`, `pickup_type`, `drop_off_type`, `` |
```
Comment (tzujenchanmbd) [inline docs/documentation/schedule/components/base.md:55]:
```suggestion
|[stop_times.txt](/schedule/reference/#stop_timestxt)|`trip_id`, `arrival_time`, `departure_time`, `stop_id`, `stop_sequence`, `pickup_type`, `drop_off_type`, `timepoint` |
```

**Review by tzujenchanmbd — CHANGES_REQUESTED** [review]
lgtm except 2 tiny editorial changes

**Comment by isabelle-dr** [comment]
Fixed :) 
PTAL

**Review by tzujenchanmbd — APPROVED** [review]

**Pull request merged**